src/main/java/uk/gov/companieshouse/siccode/api/search/CombinedSicActivitiesStorageModel.java

```diff
@@ -104,48 +108,62 @@ public int hashCode() {
         result = prime * result
                 + ((activityDescriptionSearchField == null) ? 0 : activityDescriptionSearchField.hashCode());
         result = prime * result + ((id == null) ? 0 : id.hashCode());
-        result = prime * result + (isCompaniesHouseactivity ? 1231 : 1237);
+        result = prime * result + (companiesHouseactivity ? 1231 : 1237);
         result = prime * result + ((sicCode == null) ? 0 : sicCode.hashCode());
         result = prime * result + ((sicDescription == null) ? 0 : sicDescription.hashCode());
         return result;
     }
 
     @Override
     public boolean equals(Object obj) {
-        if (this == obj)
+        if (this == obj){
```

Comment:
Another comment related to styling is the braces after if statements should include a space as per the java [styleguides](https://github.com/companieshouse/styleguides/blob/master/java.md) (See 3.): [horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)

This styling should be consistent across all classes